package/package.json

```diff
@@ -2,23 +2,13 @@
   "name": "@forgerock/login-widget",
   "type": "module",
   "exports": {
-    "inline": {
-      "import": "./inline.js",
-      "require": "./inline.cjs"
-    },
-    "modal": {
-      "import": "./modal.js",
-      "require": "./modal.cjs"
-    }
+    "import": "./index.js",
+    "require": "./index.cjs"
```

Comment:
Since we now have a single index file, I'm assuming we can remove the intermediate object with an "index" key.